CMakeLists.txt

```diff
@@ -1,6 +1,8 @@
 # ########## Project setup ##########
-PROJECT(wildmidi C)
 CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11)
+CMAKE_POLICY(SET CMP0072 NEW)
```

Comment:
Instead of setting the policy, could we just bump our cmake requirement?

Reply:
What do we have to do with FindOpenGL() to need this CMP0072 policy setting? (see: https://cmake.org/cmake/help/v3.11/policy/CMP0072.html)

Reply:
We're not an OpenGL consumer, so we shouldn't need to bother with this nor set a policy for it.

Reply:
We do not use: `FindOpenGL`

Reply:
Well, that's exactly the reason I asked, and it seems like the actual author lost interest in his own pull request..

Reply:
Sorry for the radio silence! It's been very busy at work, but I'll try to find time this week to get this PR ready!